src/main/daml/Daml/Finance/Interface/Instrument/Bond/Callable/Types.daml

```diff
@@ -29,7 +29,14 @@ data Callable = Callable
       -- ^ The minimum coupon rate possible. For example, if "3M Libor + 0.5%" would result in a
       --   rate of -0.2%, but floorRate is 0.0%, the coupon rate used would be 0.0%.
     periodicSchedule : PeriodicSchedule
```

Comment:
Shall we rename this to

```suggestion
    couponSchedule : PeriodicSchedule
```
?

Reply:
I was considering that, but the problem is that it is not only used for the coupons but also to indicate issue date / maturity date of the bond, and to generally drive the calculations. In FpML swap terms, it is both the calculationSchedule and the paymentSchedule.

Reply:
Understood, thanks. I would still prefer to use the more descriptive name, but if we prefer to use `periodicSchedule` that I would add the above details to the description (that this is both calculation and payment schedule)

Reply:
That said, doesn't `couponSchedule` encompass both `calculation` and `payment`?

Reply:
I decided to rename it and also to add an extra comment to clarify what it is used for.